File: src/constants/lint-staged.ts
import type { PackageManager } from '@/types/package-manger'

// TODO: Add configuration for yarn
export const LINT_STAGED_CONFIG: Record<PackageManager, string> = {
  npm: 'npx lint-staged',
  pnpm: 'pnpm dlx lint-staged',
  yarn: '',
  bun: 'bunx lint-staged'
}

File: internal/constants/lint_staged.go
package constants

import "github.com/RaulCatalinas/HuskyBC/internal/enums"

var LINT_STAGED_CONFIG = map[enums.PackageManager]string{
	enums.PackageManagerNpm:  "npx lint-staged",
	enums.PackageManagerYarn: "yarn dlx lint-staged",
	enums.PackageManagerPnpm: "pnpm dlx lint-staged",
	enums.PackageManagerBun:  "bunx lint-staged",
}
